1.0.7: test suite is failing (wrong path to .pem files)

<details>

```
+ /usr/bin/make -O -j48 V=1 VERBOSE=1 check
Making check in workloads
make[1]: Nothing to be done for 'check'.
Making check in src
make[1]: Nothing to be done for 'check'.
Making check in tests
/usr/bin/make  check-TESTS
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
XFAIL: unknown_proto.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
XFAIL: parse_err.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: 01simple_tcp.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: 02two_groups.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: accept-tcp.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: canfail.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: accept-connect.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: friendliness.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: disconnect_iter.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: test_4groups.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: high_connection_count.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: test_netperf.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: test-sendfile.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: test-sendfilev.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: test_send_recv.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: test-rate.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: a.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: test-ssize-iperf.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: mix_thr_proc.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: test-sendfilev-chunked.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: test-duration-process.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: 3proto.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: multi_proto_connect.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: test-ssize-flowop-count-dur.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: 01simple_sctp.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: accept-sctp.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: throughput_sctp.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: 02_2proto1group.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: 01simple_ssl.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: 01simple_udp.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
FAIL: test_udp.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.7/tests'
=======================================
   uperf 1.0.7: tests/test-suite.log
=======================================

# TOTAL: 31
# PASS:  0
# SKIP:  0
# XFAIL: 2
# FAIL:  29
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

XFAIL: unknown_proto
====================

XFAIL unknown_proto.xml (exit status: 1)

XFAIL: parse_err
================

XFAIL parse_err.xml (exit status: 1)

FAIL: 01simple_tcp
==================

FAIL 01simple_tcp.xml (exit status: 1)

FAIL: 02two_groups
==================

FAIL 02two_groups.xml (exit status: 1)

FAIL: accept-tcp
================

FAIL accept-tcp.xml (exit status: 1)

FAIL: accept-connect
====================

FAIL accept-connect.xml (exit status: 1)

FAIL: canfail
=============

FAIL canfail.xml (exit status: 1)

FAIL: disconnect_iter
=====================

FAIL disconnect_iter.xml (exit status: 1)

FAIL: friendliness
==================

FAIL friendliness.xml (exit status: 1)

FAIL: high_connection_count
===========================

FAIL high_connection_count.xml (exit status: 1)

FAIL: test_4groups
==================

FAIL test_4groups.xml (exit status: 1)

FAIL: test_netperf
==================

FAIL test_netperf.xml (exit status: 1)

FAIL: test-sendfilev-chunked
============================

FAIL test-sendfilev-chunked.xml (exit status: 1)

FAIL: test-sendfile
===================

FAIL test-sendfile.xml (exit status: 1)

FAIL: test_send_recv
====================

FAIL test_send_recv.xml (exit status: 1)

FAIL: test-rate
===============

FAIL test-rate.xml (exit status: 1)

FAIL: test-sendfilev
====================

FAIL test-sendfilev.xml (exit status: 1)

FAIL: mix_thr_proc
==================

FAIL mix_thr_proc.xml (exit status: 1)

FAIL: test-duration-process
===========================

FAIL test-duration-process.xml (exit status: 1)

FAIL: test-ssize-flowop-count-dur
=================================

FAIL test-ssize-flowop-count-dur.xml (exit status: 1)

FAIL: test-ssize-iperf
======================

FAIL test-ssize-iperf.xml (exit status: 1)

FAIL: 01simple_sctp
===================

FAIL 01simple_sctp.xml (exit status: 1)

FAIL: 3proto
============

FAIL 3proto.xml (exit status: 1)

FAIL: accept-sctp
=================

FAIL accept-sctp.xml (exit status: 1)

FAIL: multi_proto_connect
=========================

FAIL multi_proto_connect.xml (exit status: 1)

FAIL: throughput_sctp
=====================

FAIL throughput_sctp.xml (exit status: 1)

FAIL: 02_2proto1group
=====================

FAIL 02_2proto1group.xml (exit status: 1)

FAIL: a
=======

FAIL a.xml (exit status: 1)

FAIL: 01simple_ssl
==================

FAIL 01simple_ssl.xml (exit status: 1)

FAIL: 01simple_udp
==================

FAIL 01simple_udp.xml (exit status: 1)

FAIL: test_udp
==============

FAIL test_udp.xml (exit status: 1)

============================================================================
Testsuite summary for uperf 1.0.7
============================================================================
# TOTAL: 31
# PASS:  0
# SKIP:  0
# XFAIL: 2
# FAIL:  29
# XPASS: 0
# ERROR: 0
============================================================================
See tests/test-suite.log
Please report to realneel@gmail.com
============================================================================
```
In tests/log file is possible to find:
```
h=localhost duration=10s ../src/uperf -m ./unknown_proto.xml
h=localhost duration=10s ../src/uperf -m ./parse_err.xml
h=localhost duration=10s ../src/uperf -m ./01simple_tcp.xml
h=localhost duration=10s ../src/uperf -m ./02two_groups.xml
h=localhost duration=10s ../src/uperf -m ./accept-tcp.xml
h=localhost duration=10s ../src/uperf -m ./canfail.xml
Error 1: Protocol xxx not supported
Error parsing ./unknown_proto.xml
h=localhost duration=10s ../src/uperf -m ./accept-connect.xml
h=localhost duration=10s ../src/uperf -m ./disconnect_iter.xml
Error 1: Unknown symbol: <goup
h=localhost duration=10s ../src/uperf -m ./test_4groups.xml
Error parsing ./parse_err.xml
h=localhost duration=10s ../src/uperf -m ./high_connection_count.xml
h=localhost duration=10s ../src/uperf -m ./friendliness.xml
h=localhost duration=10s ../src/uperf -m ./test_netperf.xml
h=localhost duration=10s ../src/uperf -m ./test-sendfile.xml
h=localhost duration=10s ../src/uperf -m ./test-rate.xml
h=localhost duration=10s ../src/uperf -m ./test-sendfilev-chunked.xml
h=localhost duration=10s ../src/uperf -m ./test_send_recv.xml
h=localhost duration=10s ../src/uperf -m ./test-sendfilev.xml
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
h=localhost duration=10s ../src/uperf -m ./test-duration-process.xml
h=localhost duration=10s ../src/uperf -m ./test-ssize-iperf.xml
h=localhost duration=10s ../src/uperf -m ./3proto.xml
h=localhost duration=10s ../src/uperf -m ./a.xml
h=localhost duration=10s ../src/uperf -m ./multi_proto_connect.xml
h=localhost duration=10s ../src/uperf -m ./mix_thr_proc.xml
h=localhost duration=10s ../src/uperf -m ./01simple_sctp.xml
h=localhost duration=10s ../src/uperf -m ./test-ssize-flowop-count-dur.xml
h=localhost duration=10s ../src/uperf -m ./accept-sctp.xml
h=localhost duration=10s ../src/uperf -m ./throughput_sctp.xml
h=localhost duration=10s ../src/uperf -m ./test_udp.xml
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
h=localhost duration=10s ../src/uperf -m ./02_2proto1group.xml
h=localhost duration=10s ../src/uperf -m ./01simple_udp.xml
h=localhost duration=10s ../src/uperf -m ./01simple_ssl.xml
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
stat:/usr/share/uperf/server.pem:No such file or directory
Error connecting to localhost

** TCP: Cannot connect to localhost:20000 Connection refused
```
</details>

So looks like test suite is using paths to pem files not from source tree but from /usr/share/uperf/server.

Even with started `uperf -s` test suite still fails
```console
+ cd uperf-1.0.8
+ /home/tkloczko/rpmbuild/BUILDROOT/uperf-1.0.8-2.fc35.x86_64/usr/bin/uperf -s
Can't load client.pem
+ sleep 5
+ /usr/bin/make -O -j48 V=1 VERBOSE=1 check
Making check in workloads
make[1]: Nothing to be done for 'check'.
Making check in src
make[1]: Nothing to be done for 'check'.
Making check in tests
/usr/bin/make  check-TESTS
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
XFAIL: parse_err.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
XFAIL: unknown_proto.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
FAIL: 01simple_vsock.vsock.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
FAIL: test_vsock.vsock.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
FAIL: accept-sctp.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
Error creating ports: Protocol not supported
Error creating ports: Protocol not supported
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
FAIL: 01simple_sctp.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
FAIL: 3proto.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
Error creating ports: Protocol not supported
Error creating ports: Protocol not supported
Error creating ports: Protocol not supported
Error creating ports: Protocol not supported
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
FAIL: a.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
FAIL: 02_2proto1group.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
FAIL: throughput_sctp.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
FAIL: multi_proto_connect.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
** Warning: Send buffer: 4.50KB (Requested:2.00KB)
** Warning: Recv buffer: 4.00KB (Requested:2.00KB)
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: 01simple_tcp.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
** 11 Warnings Send buffer: 400.00KB (Requested:200.00KB)
** 11 Warnings Recv buffer: 400.00KB (Requested:200.00KB)
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: test_send_recv.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
** Warning: Send buffer: 4.50KB (Requested:2.00KB)
** Warning: Recv buffer: 4.00KB (Requested:2.00KB)
** Warning: Send buffer: 4.50KB (Requested:2.00KB)
** Warning: Recv buffer: 4.00KB (Requested:2.00KB)
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
FAIL: 01simple_ssl.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: 01simple_udp.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
** 6 Warnings Send buffer: 400.00KB (Requested:200.00KB)
** 6 Warnings Recv buffer: 400.00KB (Requested:200.00KB)
** 8 Warnings Send buffer: 400.00KB (Requested:200.00KB)
** 8 Warnings Recv buffer: 400.00KB (Requested:200.00KB)
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: accept-connect.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: accept-tcp.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: disconnect_iter.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
** 4 Warnings Send buffer: 100.00KB (Requested:50.00KB)
** 4 Warnings Recv buffer: 100.00KB (Requested:50.00KB)
** 32 Warnings Send buffer: 100.00KB (Requested:50.00KB)
** 32 Warnings Recv buffer: 100.00KB (Requested:50.00KB)
** 200 Warnings Send buffer: 400.00KB (Requested:200.00KB)
** 200 Warnings Recv buffer: 400.00KB (Requested:200.00KB)
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: high_connection_count.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
** 128 Warnings Send buffer: 100.00KB (Requested:50.00KB)
** 128 Warnings Recv buffer: 100.00KB (Requested:50.00KB)
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
** Warning: Send buffer: 400.00KB (Requested:200.00KB)
** 4 Warnings Send buffer: 400.00KB (Requested:200.00KB)
PASS: test-ssize-flowop-count-dur.xml
** Warning: Recv buffer: 400.00KB (Requested:200.00KB)
** 4 Warnings Recv buffer: 400.00KB (Requested:200.00KB)
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
** 8 Warnings Send buffer: 400.00KB (Requested:200.00KB)
** 8 Warnings Recv buffer: 400.00KB (Requested:200.00KB)
** 8 Warnings Send buffer: 400.00KB (Requested:200.00KB)
** 8 Warnings Recv buffer: 400.00KB (Requested:200.00KB)
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: 02two_groups.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: mix_thr_proc.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
** 4 Warnings Send buffer: 100.00KB (Requested:50.00KB)
** 4 Warnings Recv buffer: 100.00KB (Requested:50.00KB)
** 4 Warnings Send buffer: 100.00KB (Requested:50.00KB)
** 4 Warnings Recv buffer: 100.00KB (Requested:50.00KB)
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: test-sendfilev-chunked.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: test-sendfilev.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
** Warning: Send buffer: 400.00KB (Requested:200.00KB)
** Warning: Recv buffer: 400.00KB (Requested:200.00KB)
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: test_netperf.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
** 24 Warnings Send buffer: 416.00KB (Requested:256.00KB)
** 24 Warnings Recv buffer: 416.00KB (Requested:256.00KB)
** 24 Warnings Send buffer: 416.00KB (Requested:256.00KB)
** 24 Warnings Recv buffer: 416.00KB (Requested:256.00KB)
** 24 Warnings Send buffer: 416.00KB (Requested:256.00KB)
** 24 Warnings Recv buffer: 416.00KB (Requested:256.00KB)
** 24 Warnings Send buffer: 416.00KB (Requested:256.00KB)
** 24 Warnings Recv buffer: 416.00KB (Requested:256.00KB)
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: test_4groups.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: max_thread_count.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: test-sendfile.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: test-ssize-iperf.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: test-rate.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: test_udp.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
** 12 Warnings Send buffer: 416.00KB (Requested:500.00KB)
** 12 Warnings Recv buffer: 416.00KB (Requested:500.00KB)
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: test-duration-process.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: max_procs_count.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
** Warning: Send buffer: 400.00KB (Requested:200.00KB)
** Warning: Recv buffer: 400.00KB (Requested:200.00KB)
** Warning: Send buffer: 400.00KB (Requested:200.00KB)
** Warning: Recv buffer: 400.00KB (Requested:200.00KB)
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: friendliness.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
** 10001 Warnings Send buffer: 400.00KB (Requested:200.00KB)
** 10001 Warnings Recv buffer: 400.00KB (Requested:200.00KB)
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
PASS: canfail.xml
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[3]: Entering directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
=======================================
   uperf 1.0.8: tests/test-suite.log
=======================================

# TOTAL: 35
# PASS:  23
# SKIP:  0
# XFAIL: 2
# FAIL:  10
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

XFAIL: unknown_proto
====================

XFAIL unknown_proto.xml (exit status: 1)

XFAIL: parse_err
================

XFAIL parse_err.xml (exit status: 1)

FAIL: 01simple_sctp
===================

FAIL 01simple_sctp.xml (exit status: 1)

FAIL: 3proto
============

FAIL 3proto.xml (exit status: 1)

FAIL: accept-sctp
=================

FAIL accept-sctp.xml (exit status: 1)

FAIL: multi_proto_connect
=========================

FAIL multi_proto_connect.xml (exit status: 1)

FAIL: throughput_sctp
=====================

FAIL throughput_sctp.xml (exit status: 1)

FAIL: 02_2proto1group
=====================

FAIL 02_2proto1group.xml (exit status: 1)

FAIL: a
=======

FAIL a.xml (exit status: 1)

FAIL: 01simple_ssl
==================

FAIL 01simple_ssl.xml (exit status: 1)

FAIL: 01simple_vsock.vsock
==========================

FAIL 01simple_vsock.vsock.xml (exit status: 1)

FAIL: test_vsock.vsock
======================

FAIL test_vsock.vsock.xml (exit status: 1)

============================================================================
Testsuite summary for uperf 1.0.8
============================================================================
# TOTAL: 35
# PASS:  23
# SKIP:  0
# XFAIL: 2
# FAIL:  10
# XPASS: 0
# ERROR: 0
============================================================================
See tests/test-suite.log
Please report to realneel@gmail.com
============================================================================
make[3]: *** [Makefile:489: test-suite.log] Error 1
make[3]: Leaving directory '/home/tkloczko/rpmbuild/BUILD/uperf-1.0.8/tests'
make[2]: *** [Makefile:597: check-TESTS] Error 2
make[1]: *** [Makefile:662: check-am] Error 2
make: *** [Makefile:354: check-recursive] Error 1
```

--- a/.github/workflows/c-cpp.yml
+++ b/.github/workflows/c-cpp.yml
@@ -25,9 +25,5 @@ jobs:
       run: ./configure
     - name: make
       run: make
-    - name: start localhost server
-      run: ./src/uperf -s &
-    - name: start localhost server for vsock
-      run: ./src/uperf -s -S vsock &
     - name: make check
       run: make check

--- a/tests/test.sh
+++ b/tests/test.sh
@@ -12,7 +12,24 @@ if [[ "$profile" == *".vsock.xml" ]]; then
     fi
 fi
 
+# Start server
+serverpid=""
+if pgrep uperf ; then
+    echo "uperf server already running; please stop it and try again"
+    exit
+else
+    echo "Starting server - $uperf -s $csocket"
+    $uperf -s $csocket &
+    serverpid=$!
+fi
+
 echo h=$host duration=10s $uperf $csocket -m $profile >>log
 h=$host duration=10s $uperf $csocket -m $profile >> log 2>&1
+exitstatus=$?
 
-
+# kill server
+if [[ serverpid != "" ]] ; then
+   echo "Killing $serverpid"
+   kill -9 $serverpid
+fi
+exit $exitstatus

